tests/fsharp/SDKTests/tests/DefaultFSharpCore - IsNotPreviewSdk.proj

```diff
@@ -0,0 +1,20 @@
+<Project ToolsVersion="4.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Verifies that in a non preview sdk it selects the recently built fsharp.core package -->
```

Comment:
I thought a non-preview sdk should use the boring, shipped version, not the just built one?